railties/lib/rails/generators/rails/app/templates/env.erb.tt

```diff
@@ -0,0 +1,5 @@
+<%%# This env template should lookup values from a password store or from ENV -%>
+# Generated by kamal envify from .env.erb
+KAMAL_REGISTRY_PASSWORD=<%%= "YOUR-KEY-FROM-PASSWORD-STORE" %>
+RAILS_MASTER_KEY=<%%= File.read("config/master.key") %>
+# GITHUB_TOKEN=<%%#= `gh config get -h github.com oauth_token`.strip %>
```

Comment:
This is a neat idea but can we can rely on having the `gh` cli?

Reply:
No, which is why it's commented out by default. So it's just a suggestion for how to do this.

Reply:
🤦‍♂️Ofcourse. Missed the comment without the highlighting.